### DISCUSSION TITLE
Is stream reading possible in firefox?

### DISCUSSION THREAD

**Discussion by wannacfuture** [wannacfuture/PsW#4] [Q&A] [answered]
I just wanna know if stream reading is possible in firefox.

**Comment by sweetapplepumpkinpie** [comment, marked as answer]
Yes, stream reading is possible in Firefox. The ReadableStream API, which allows you to read data from a stream, is supported in Firefox since version 65. You can use it to read data from a stream, such as a file, a response body, or any other custom stream.
